port/oc_log_internal.h

```diff
@@ -268,9 +270,15 @@
   do {                                                                         \
     char *beg = (buff);                                                        \
     char *end = (buff) + (size);                                               \
-    for (size_t i = 0; beg <= (end - 3) && i < (size_t)(len); i++) {           \
-      beg += (i == 0) ? SPRINTF(beg, "%02x", (data)[i])                        \
-                      : SPRINTF(beg, ":%02x", (data)[i]);                      \
+    /* without _oc_written = 9 has sometimes */                                \
```

Comment:
`_oc_written`? is that renamed `oc_log_ret`?

Reply:
yes - fixed